### PR TITLE
INT Demo fixes

### PR DESCRIPTION
**Pull request by herlenashavi**
1) Makefile.bmv2 added to build switch with switchlink
2) Changes to docker.mk to build bmv2 docker image
3) Changes to install_deps.sh - ibnanomsg dependency install 
4) Changes to Dockerfile
5) Updated INT readme
6) Removed switch network level tests moved to switch repo

### CODE REVIEW

**Comment by herlenashavi** [comment]
1) Removed Makefile.bmv2
2) Readme changes
3) INT script changes to take model-dir as parameter
4) docker.mk changes to take installpath as parameter

File: docker/Dockerfile [hunk 1]
```diff
@@ -37,17 +43,10 @@ RUN apt-get install -y \
     redis-tools \
     subversion \
     tshark \
+    vim \
```
Comment (antoninbas) [inline docker/Dockerfile]: are `vim` and `autopoint` needed?

Comment (cruzj) [inline docker/Dockerfile]: Isn't VI installed by default?

Comment (herlenashavi) [inline docker/Dockerfile]: Removed autopoint and vim from Dockerfile.

**Pull request merged**